user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/GQI/Query_Data_Sources/Get_resources.md

```diff
@@ -0,0 +1,17 @@
+---
+uid: Get_resources
+---
+
+# Get resources
+
+Available from DataMiner 10.2.8/10.3.0 onwards<!--RN 33553-->. The *Get resources* data source (...)
```

Comment:
I have made some changes based on your feedback, @AndreRodriguesSkyline. Could you complete this sentence, specifying in short what this data source does? Example: [Get alarms data source](https://docs.dataminer.services/user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/GQI/Query_Data_Sources/Get_alarms.html). Thanks in advance!

Reply:
I that "The *Get resources* data source retrieves all resources in the DataMiner system" should be fine.

Reply:
Done in commit [d0880c7](https://github.com/SkylineCommunications/dataminer-docs/pull/2090/commits/d0880c78c049f4fcec09e64d2ffbc9c19da2d975). Thank you!